ansible/roles/postgresql/templates/pg_hba.conf.j2

```diff
@@ -99,6 +99,9 @@ host    all             all             ::1/128                 md5
 # Allow replication connections from localhost, by a user with the
 # replication privilege.
 local   replication     postgres                                peer
+{% if hot_standby_server|default(False) %}
+host    replication     postgres        {{ hot_standby_server }}/32 trust
```

Comment:
If this get's changed from `trust` to `md5` where would we put the password for pg_backrest?

Reply:
Just trying to figure out how we merge this with https://github.com/dimagi/commcarehq-ansible/pull/962/files#diff-d959b4d4814d1bc5d1d095125136a90aR105

Reply:
From the docs "It is possible to configure a password in the primary_conninfo setting but using a .pgpass file is more flexible and secure."
But our current pgbackrest setup doesn't need it, I used this to be able to remotely pg_basebackup, instead of doing it locally and then rsync (saves an step).

Reply:
it's ok to remove this or change to md5